src/community/design-system-day-2024/index.md

```diff
@@ -12,9 +12,11 @@ order: 15
 <img class="app-image--no-border govuk-!-margin-bottom-9" src="/images/dsd24-24.svg" alt="" role="presentation">
 
 <p class="govuk-!-font-size-24">
-We had a great response to the call for speakers for Design System Day 2024 in Liverpool. We’re grateful to our community for all the effort you put into helping us make these events brillant.  
+Design System Day is coming to Liverpool!  
 </p>
 
+The event will be taking place on Thursday 5 September 2024 at St George's Hall. Tickets for the event have sold out however all talks will be recorded and shared later in the year. Sign up to our [mailing list](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system) to be notified when they become available.
```

Comment:
Making this suggestion about link text so that it's a bit more descriptive out of context for screen reader users.

```suggestion
The event will be taking place on Thursday 5 September 2024 at St George's Hall. Tickets for the event have sold out however all talks will be recorded and shared later in the year. [Sign up to our mailing list](https://mailchi.mp/707ce8dec373/get-updated-by-email-govuk-design-system) to be notified when they become available.
```